en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
-- `Yaw scale`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
+- `Yaw Torque`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
 - `Trim`: An offset added to the actuator so that it is centered without input.
   This might be determined by trial and error.
-- `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
+- (Advanced) `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
   - The setting limits the rate of change of an actuator (if not specified then no rate limit is applied).
      It is intended for actuators that may be damaged if they move too fast — such as the tilting actuators on a tiltrotor VTOL vehicle.
   - For example, a setting of 2.0 means that the motor/servo will not be commanded to move from 0 to 1 at a rate that completes the operation in less than 2 seconds (in case of reversible motors, the range is -1 to 1).
-- `Lock control surfaces in hover`:
+- (Advanced) `Flap Scale`: How much this actuator is deflected at the "full flaps configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as flap or to compensate for generated torque through main flaps.
+- (Advanced) `Spoiler Scale`: How much this actuator is deflected at the "full spoiler configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as spoiler or to compensate for generated torque through main spoiler.
+- (VTOL only) `Lock control surfaces in hover`:
   - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
   - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
 
+#### Flaps and Spoilers Configuration
+
+`Flap-control` and `spoiler-control` are aerodynamic configurations that can either be commanded manually by the pilot through a switch on the RC, or are set automatically by the controller, e.g. to reduce the airspeed before landing. 
+The flap and spoiler configuration controls are of the range of [0, 1] and can then be arbitrarily allocated to the control surfaces.
+
+It's important to understand the difference: with `flap/spoiler-control` we refer to an abstract metric of "how much flap/spoiler configuration" the controller currently demands, while the actual flaps or spoiler actuator setpoint can vary from that value.
```

Comment:
Let me pull this apart and just talk about spoilers. Because the terms configuration and control are scattered here and so ambigous.

> `Flap-control` and `spoiler-control` are aerodynamic configurations that can either be commanded manually by the pilot through a switch on the RC, or are set automatically by the controller, e.g. to reduce the airspeed before landing. 

The pilot or come controller says "I want to set spoiler control" because I am landing. 

> The flap and spoiler configuration controls are of the range of [0, 1] and can then be arbitrarily allocated to the control surfaces.

Or are you saying there is an abstract spoiler control setting that has a range [0,1] that a controller can request. SO say it requests 50% you are saying that 50% is the "abstract setpoint" but the allocator can take this 50% and assign it to different control surfaces?

> It's important to understand the difference: with `flap/spoiler-control` we refer to an abstract metric of "how much flap/spoiler configuration" the controller currently demands, while the actual flaps or spoiler actuator setpoint can vary from that value.

This is still confusing but again, I think you're saying that "spoiler" control is the abstract amount - say 30% of full. This is like the "abstract spoiler actuator setpoint". However in reality the behaviour is spread across a bunch of controls, so there is no one "spoiler actuator setpoint" but more a bunch"

Reply:
Excellent. I'll reword tomorrow. Note comment https://github.com/PX4/PX4-user_guide/pull/2450#discussion_r1170850045